packages/rpc-core/src/types/rpc-methods/getBlocks.d.ts

```diff
@@ -0,0 +1,18 @@
+import { IJsonRpcTransport } from '@solana/rpc-transport';
+
+type GetBlocksApiResponse = Slot[];
+
+declare interface GetBlocksApi {
+    /**
+     * Returns a list of confirmed blocks between two slots
+     */
+    getBlocks(
+        transport: IJsonRpcTransport,
+        startSlot: Slot,
+        endSlotInclusive?: Slot,
+        config?: readonly {
+            // Defaults to `finalized`
+            commitment?: Exclude<Finality, 'processed'>;
+        }
+    ): Promise<GetBlocksApiResponse>;
```

Comment:
See https://docs.solana.com/api/http#getblocks.

Reply:
I fixed it! https://github.com/solana-labs/solana/pull/30351/files

Reply:
Ah… https://discord.com/channels/428295358100013066/966782389839876096/1083467864650039347

Reply:
ah yes. Anyway, thanks for fixing!